incubator/kube-downscaler/templates/deployment.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     helm.sh/chart: {{ include "kube-downscaler.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app: {{ include "kube-downscaler.name" . }}
```

Comment:
why not just use app.kubernetes.io/name?

Reply:
@prydonius it's common to add this as it's shorter to type. I saw it in lots of charts

Reply:
It's in a lot of existing charts for historical reasons (before the [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)), but if you're already using `app.kubernetes.io/name` I don't really see the need to add this, apart from it being faster to type. I also can't see any charts that use both `app.kubernetes.io/name` and `app` today.